README.md

```diff
@@ -131,138 +160,161 @@ pre_commit_db_rm_repo () {  # Requires sqlite3
-```
+
 **Rationale:** cf. http://stackoverflow.com/a/38439681/636849
 
+```yaml
+- repo: local
+  hooks:
+    - id: Jenkinsfile-linter
+      name: Check Jenkinsfile following the scripted-pipeline syntax using Jenkins
+        API
+      files: Jenkinsfile
+      language: system
+      entry: sh -c '! curl --silent $JENKINS_URL/job/MyPipelineName/job/master/1/replay/checkScriptCompile
+        --user $JENKINS_USER:$JENKINS_TOKEN --data-urlencode value@Jenkinsfile |
+        grep -F "\"status\":\"fail\""'
 ```
--   repo: local
-    hooks:
-        -   id: Jenkinsfile-linter
-            name: Check Jenkinsfile following the scripted-pipeline syntax using Jenkins API
-            files: Jenkinsfile
-            language: system
-            entry: sh -c '! curl --silent $JENKINS_URL/job/MyPipelineName/job/master/1/replay/checkScriptCompile --user $JENKINS_USER:$JENKINS_TOKEN --data-urlencode value@Jenkinsfile | grep -F "\"status\":\"fail\""'
-```
-Note: the `$JENKINS_TOKEN` can be retrieved from `$JENKINS_URL/user/$USER_NAME/configure`
 
-Beware, in 1 case on 6 I faced this unsolved bug with explictely-loaded libraries: https://issues.jenkins-ci.org/browse/JENKINS-42730
+Note: the `$JENKINS_TOKEN` can be retrieved from
+`$JENKINS_URL/user/$USER_NAME/configure`
+
+Beware, in 1 case on 6 I faced this unsolved bug with explictly-loaded
+libraries: https://issues.jenkins-ci.org/browse/JENKINS-42730 .
 
-Also, there is also a linter for the declarative syntax: https://jenkins.io/doc/book/pipeline/development/#linter
+Also, there is also a linter for the declarative syntax:
+https://jenkins.io/doc/book/pipeline/development/#linter .
 
 ### Forbid some Javascript keywords for browser retrocompatibility issues
 
-    -   repo: local
-        hooks:
-        -   id: js-forbid-const
-            name: The const keyword is not supported by IE10
-            language: pygrep
-            entry: "const "
-            files: \.js$
-        -   id: js-forbid-let
-            name: The let keyword is not supported by IE10
-            language: pygrep
-            entry: "let "
-            files: \.js$
+```yaml
+- repo: local
+  hooks:
+    - id: js-forbid-const
+      name: The const keyword is not supported by IE10
+      language: pygrep
+      entry: 'const '
+      files: \.js$
+    - id: js-forbid-let
+      name: The let keyword is not supported by IE10
+      language: pygrep
+      entry: 'let '
+      files: \.js$
+```
 
 ### CSS
 
-    -   repo: local
-        hooks:
-        -   id: css-forbid-px
-            name: In CSS files, use rem or % over px
-            language: pygrep
-            entry: px
-            files: \.css$
-        -   id: ot-sanitize-fonts
-            name: Calling ot-sanitise on otf/ttf/woff/woff2 font files
-            language: system
-            entry: sh -c 'type ot-sanitise >/dev/null && for font in "$@"; do echo "$font"; ot-sanitise "$font"; done || echo "WARNING Command ot-sanitise not found - skipping check"'
```

Comment:
The line breaks seem a bit akward but that's a detail

Reply:
I apply a script (manually) to fix minor deviations from a good formatted yaml and this also limits line lengths.  Line splits are not always in logical locations, but long lines are not more readable IMHO.

Reply:
I've update the line splits a bit in the `README.md`.